Skip xc cli tests, 1.8.3 failed when run with no arguments for these commands
but 1.8.4 doesn't.

--- gopass-1.8.4/src/github.com/gopasspw/gopass/commands_test.go
+++ gopass-1.8.4/src/github.com/gopasspw/gopass/commands_test.go
@@ -55,14 +55,14 @@
 	".templates.remove":      {},
 	".templates.show":        {},
 	".unclip":                {},
-	".xc.decrypt":            {},
-	".xc.encrypt":            {},
-	".xc.export":             {},
-	".xc.export-private-key": {},
-	".xc.generate":           {},
-	".xc.import":             {},
-	".xc.import-private-key": {},
-	".xc.remove":             {},
+	// ".xc.decrypt":            {},
+	// ".xc.encrypt":            {},
+	// ".xc.export":             {},
+	// ".xc.export-private-key": {},
+	// ".xc.generate":           {},
+	// ".xc.import":             {},
+	// ".xc.import-private-key": {},
+	// ".xc.remove":             {},
 }
 
 func TestGetCommands(t *testing.T) {
